assets/svg/award.svg

```diff
@@ -0,0 +1 @@
/www.w3.org/2000/svg" viewBox="0 0 400 400"><defs><style>.cls-1,.cls-11,.cls-12,.cls-3,.cls-4,.cls-5,.cls-7,.cls-8,.cls-9{fill:none;}.cls-1{stroke:#e8eaed;stroke-linecap:round;}.cls-1,.cls-11,.cls-7,.cls-8,.cls-9{stroke-miterlimit:10;}.cls-1,.cls-11,.cls-7,.cls-9{stroke-width:6px;}.cls-2{fill:#f1f3f4;}.cls-3{stroke:#fbbc04;}.cls-12,.cls-3,.cls-4,.cls-5{stroke-linejoin:round;stroke-width:9px;}.cls-4{stroke:#f29900;}.cls-5{stroke:#ea8600;}.cls-6{fill:#f9ab00;}.cls-7{stroke:#fff;}.cls-8,.cls-9{stroke:#bdc1c6;}.cls-8{stroke-width:20px;}.cls-10{fill:#dadce0;}.cls-11{stroke:#f1f3f4;}.cls-12{stroke:#fcc934;}</style></defs><line class="cls-1" x1="382" y1="94.2" x2="361.09" y2="94.2"/><line class="cls-1" x1="364.25" y1="33.77" x2="346.66" y2="45.07"/><line class="cls-1" x1="197.01" y1="9.74" x2="210.71" y2="25.54"/><line class="cls-1" x1="162.97" y1="62.74" x2="183.03" y2="68.62"/><line class="cls-1" x1="162.98" y1="125.72" x2="183.04" y2="119.83"/><line class="cls-1" x1="316.7" y1="195.86" x2="308.01" y2="176.84"/><line class="cls-1" x1="364.28" y1="154.59" x2="346.69" y2="143.29"/><ellipse class="cls-2" cx="154.11" cy="381.74" rx="103.5" ry="8.26"/><path class="cls-3" d="M166.37,181.8c54.51-10.47,66-55.59,28.92-92.64"/><path class="cls-4" d="M148.8,254.55l16.54,51.35a90,90,0,0,1,2,47.76l-6.46,28.08h23"/><path class="cls-4" d="M125.3,261.45l8.46,34.92a89.84,89.84,0,0,1-.27,43.48l-10.68,41.83,20,.17"/><line class="cls-5" x1="148.8" y1="254.55" x2="158.33" y2="284.15"/><line class="cls-5" x1="128.39" y1="274.19" x2="132.8" y2="291.71"/><polygon class="cls-6" points="165.19 159.52 130.41 58.66 29.56 93.45 64.34 194.3 99.13 295.15 199.98 260.37 165.19 159.52"/><path class="cls-7" d="M71.7,138.64c16.69,24.27,52.79,17.16,59.56-16.13"/><path class="cls-7" d="M75.78,131.55A17.14,17.14,0,0,1,71.91,138a28,28,0,0,1-7.23,4.93"/><line class="cls-8" x1="269.28" y1="98.8" x2="244.62" y2="190.46"/><path class="cls-9" d="M197.37,113.58a11.4,11.4,0,1,0,1.41-22.31,38.5,38.5,0,0,1-8.49-.63,22.81,22.81,0,1,1,28-16.1"/><path class="cls-9" d="M324.07,147.67a11.4,11.4,0,1,1,10-20s4.61,2.89,7.65,3.71a22.8,22.8,0,1,0-16.09-27.95"/><path class="cls-10" d="M253.5,157.45c37.27,10,80.69-30.92,97-91.47l-135-36.32C199.23,90.21,216.23,147.42,253.5,157.45Z"/><path class="cls-11" d="M224,109.65c-5.87-19.74-6.24-43.45-1.16-67.66"/><path class="cls-11" d="M241.18,139.46a54.17,54.17,0,0,1-11.61-15.67"/><path class="cls-10" d="M248,177.83c-13.07-3.52-26,2.23-28.83,12.84L266.5,203.4C269.36,192.79,261.08,181.34,248,177.83Z"/><path class="cls-9" d="M266,110.86l-17.44,3.92a2.18,2.18,0,0,1-2.46-3l7.5-16.23-9.12-15.37a2.19,2.19,0,0,1,2.14-3.28L264.4,79l11.8-13.42a2.18,2.18,0,0,1,3.78,1l3.48,17.53,16.41,7.08a2.18,2.18,0,0,1,.2,3.91l-15.6,8.72-1.66,17.79a2.18,2.18,0,0,1-3.66,1.4Z"/><path class="cls-12" d="M74.48,209.27c74.32,45.28,152.4-1.07,170.14-18.81"/><path class="cls-12" d="M87.61,216.33a129.71,129.71,0,0,0,13.13,5.3"/><path class="cls-3" d="M74.48,209.27a136.76,136.76,0,0,0,13.13,7.06"/></svg>
```

Comment:
Let's update the formatting to be consistent with other source files. The compression and optimization should happen at build time, but the source file should be optimized for readability. That is, I would expect this to be formatted more like a normal xml document like `assets/svg/logo-g.svg`.

Also, we should check if attributes like `id="_1_Step_Darker" data-name="1 Step Darker"` are preserved when importing, and if so remove them here since they shouldn't be relevant.